src/extension/xtab/node/xtabProvider.ts

```diff
@@ -87,6 +93,9 @@ export class XtabProvider implements IStatelessNextEditProvider {
 
 	private forceUseDefaultModel: boolean = false;
 
+	private nesAcceptCounter: number = 0;
```

Comment:
Instead of absolute counts, wouldn't it make more sense to store how many % of suggestions were accepted / rejected / ignored ?

Reply:
I see. Yes that should be possible. Are you planning on passing number values to the model or a string value like: `active`, `balanced` or `passive`? We have been considering providing such a setting to the user which would lead to showing less/more suggestions and maybe slower/faster suggestions depending on the setting. Providing this information to the model would be great. 

Maybe if the user does not configure this setting, then we would try and set if for the user depending on how often they interact with NES. We could also provide a different value (`active`, `passive`, `balanced`) based on the file the user is in or reset the state when the user commits it's changes.